Content.Shared/Lathe/LatheComponent.cs

```diff
@@ -86,6 +86,12 @@ public sealed class LatheComponent : Component
         public float PartRatingMaterialUseMultiplier = DefaultPartRatingMaterialUseMultiplier;
 
         public const float DefaultPartRatingMaterialUseMultiplier = 0.85f;
+
+        /// <summary>
+        /// Checks if the lather can eject materials from it
+        /// </summary>
+        [ViewVariables(VVAccess.ReadWrite), DataField("canEject")]
+        public bool CanEject = true;
```

Comment:
This is never checked on server, only client.

Reply:
Fixed